homeassistant/components/palazzetti/config_flow.py

```diff
@@ -48,3 +53,46 @@ async def async_step_user(
             data_schema=vol.Schema({vol.Required(CONF_HOST): str}),
             errors=errors,
         )
+
+    async def async_step_dhcp(
+        self, discovery_info: dhcp.DhcpServiceInfo
+    ) -> ConfigFlowResult:
+        """Handle DHCP discovery."""
+
+        LOGGER.debug(
+            "DHCP discovery detected Palazzetti: %s", discovery_info.macaddress
+        )
+
+        self._async_abort_entries_match({CONF_HOST: discovery_info.ip})
```

Comment:
Why would it be aborted if the IP is known?

I think this can just be removed.